docs/gallery/interactive/fast_fourier_transform.ipynb

```diff
@@ -0,0 +1,264 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Definitions\n",
+    "\n",
+    "| Norm        | Equation                                                                                                 |\n",
+    "|-------------|----------------------------------------------------------------------------------------------------------|\n",
+    "| `'none'`    | --                                                                                                       |\n",
+    "| `'unitary'` | $X_{\\text{SS}}(k) = \\left\\{ \\begin{array}{ll} X(k) & \\forall k=0, k=\\frac{N}{2} \\\\ 2 X(k) & \\forall 0<k< \\frac{N}{2} \\end{array} \\right.$|\n",
+    "| `'amplitude'` | $\\overline{X}_{\\text{SS}}(k) = \\frac{1}{N} X_{\\text{SS}}(k)$                                          |\n",
+    "| `'rms'`       | $\\overline{X}_{RMS}(k) = \\left\\{ \\begin{array}{ll} \\frac{1}{\\sqrt{2}} \\overline{X}_{\\text{SS}}(k) & \\forall 0<k< \\frac{N}{2} \\\\ \\quad \\overline{X}_{\\text{SS}}(k) & \\forall k=0, k=\\frac{N}{2} \\end{array} \\right.$|\n",
+    "| `'power'`     | $\\overline{\\overline{X}}_{\\text{SS}}(k) = \\lvert \\overline{X}_{\\text{RMS}}(k) \\lvert ^2$           |\n",
+    "| `'psd'`       | $\\overline{\\overline{\\underline{X}}}_{\\text{SS}}(k) = \\frac{N}{f_s} \\overline{\\overline{X}}_{\\text{SS}}(k) = \\frac{N}{f_s} \\lvert \\overline{X}_{\\text{RMS}}(k) \\lvert ^2$|\n",
+    "\n",
+    "Note that all pyfar signals are real-valued, leading to single-sided spectra $X_{\\text{SS}}(k)$. So there are small differences in the definitions compared to the formulas written in [[1]](http://www.aes.org/e-lib/browse.cfm?elib=20838)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Explanations\n",
+    "\n",
+    "* `'none'`:\n",
+    "        Use the spectrum as it is. This norm is to be used for energy signals such as impulse responses (IRs). In this case the spectrum is independent of the signal length. For power signals, the spectrum depends on the number of samples (\"longer signal = more energy\"), so different normalizations are appropriate.\n",
+    "* `'unitary'`:\n",
+    "        Multiply the spectrum with a factor of 2 in order to represent power related measures correctly (e.g., the amplitude or RMS-value, see below). **All following normalizations make use of this.**\n",
```

Comment:
A formulation that would be true for complex signals as well would be:

```suggestion
    "        Multiply the spectrum with a factor of 2 in order to represent power related measures correctly (e.g., the amplitude or RMS-value, see below). **All following normalizations make use of this for signals real valued time signals, i.e., signals with a single-sided spectrum.**\n",
```

Reply:
Used slightly different formulation